packages/flutter/test/widgets/focus_traversal_test.dart

```diff
@@ -1474,7 +1474,7 @@ void main() {
       /// Layout is:
       ///           [0, 1]
       ///    [0, 0] [    ] [0, 2]
-      ///    [  1,  0 ] [1, 1] [1, 2]
+      ///    [1, 0] [1, 1] [1, 2]
```

Comment:
Actually, that was intentional, to show that the button wasn't the same size as the one above it.

Reply:
Ah ok, let me restore this before merging. :sweat_smile: